README.md

```diff
@@ -17,9 +17,9 @@ const DelegatedPeerRouting = require('libp2p-delegated-peer-routing')
 const routing = new DelegatedPeerRouing()
 
 try {
-  const peerInfo = await routing.findPeer('peerid')
+  const { id, addresses } = await routing.findPeer('peerid')
```

Comment:
This will fail, needs to be `addrs`.

Also, I realized this is different than the peer discovery api. Perhaps we should make peer routing consistent with peer discovery and do `{ id, multiaddrs }`? https://github.com/libp2p/js-libp2p-interfaces/tree/v0.3.x/src/peer-discovery#discoverying-peers

```suggestion
  const { id, addrs } = await routing.findPeer('peerid')
```